test/plugin/base_test.rb

```diff
@@ -1052,7 +1145,7 @@ def test_http_request_from_record
     end
     verify_log_entries(1, COMPUTE_PARAMS, 'httpRequest') do |entry|
       assert_equal http_request_message, entry['httpRequest'], entry
```

Comment:
Do we still need to override `http_request_message`? If not, let's use the constant directly instead of this method.